runhouse/main.py

```diff
@@ -798,9 +806,12 @@ def restart(
         False,
         help="Whether HTTP server started from inside a Python call rather than CLI.",
     ),
+    log_level: str = typer.Option(
```

Comment:
I think we also need to enforce that the value gets converted to uppercase - should be able to do this: 

```
log_level: str = typer.Option(
    default=DEFAULT_LOG_LEVEL, 
    help="Minimum log level for logs to be printed",
    callback=lambda value: value.upper()
)
```